### ISSUE TITLE
fitting help says chisq is normalized to number of points (Trac #930)

### ISSUE THREAD

**Issue by pkienzle** [SasView/sasview#1012]
The following sentence appears in fitting_help.rst:

  As the fit of the theory to the experimental data improves the value of 'chi2/Npts' will decrease. A good model fit should easily produce values of 'chi2/Npts' that are close to one, and certainly <100. See :ref:`Assessing_Fit_Quality`.

Check whether GUI is reporting chisq/Npts or chisq/DOF; correct docs or code and docs.

Migrated from http://trac.sasview.org/ticket/930
```json
{
    "status": "closed",
    "changetime": "2018-03-19T21:17:25",
    "_ts": "2018-03-19 21:17:25.069592+00:00",
    "description": "The following sentence appears in fitting_help.rst:\n\n  As the fit of the theory to the experimental data improves the value of 'chi2/Npts' will decrease. A good model fit should easily produce values of 'chi2/Npts' that are close to one, and certainly <100. See :ref:`Assessing_Fit_Quality`.\n\nCheck whether GUI is reporting chisq/Npts or chisq/DOF; correct docs or code and docs.\n",
    "reporter": "pkienzle",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Bug Fixing",
    "time": "2017-04-06T17:03:27",
    "component": "SasView",
    "summary": "fitting help says chisq is normalized to number of points",
    "priority": "critical",
    "keywords": "",
    "milestone": "SasView 4.2.0",
    "owner": "butler",
    "type": "defect"
}
```

**Comment by pkienzle** [comment]
Trac update at `2017/04/06 17:03:44`: **pkienzle** changed priority from "major" to "trivial"

**Comment by wpotrzebowski** [comment]
Trac update at `2017/04/07 08:52:54`: **wojciech** commented:

complete1D in fitting.py function says: 
chisqr = np.average(residuals * residuals)
where residuals=res = (fn - gn) / en
fn = data, gn = computed intensity, en = error

So in a word our chisqr is divided by number of points

**Comment by butlerpd** [comment]
Trac update at `2017/10/26 15:21:11`: **butler** commented:

I don't know where it is happening exactly but the reported chisqr on the GUI IS in fact correctly displayed as chisq/DOF not chisq/Npts.

You can check this by pulling up a data set, for example AOT_Microemulsion-Core-Contrast.xml from the test folder and fit to a sphere with 3 parameters varying: scale, background and radius. Note the Chi2/Npts reported.  Multiply this by [Npts(Fit)-DOF]/Npts(Fit).  Now press compute (same parameters as were just fit and watch the Chi2/Npts change.  Compare the new number to the one you just calculated.....  So clearly it is doing this .... somewhere.  

Would like to find where it is doing this before closing the ticket but we need to fix the docs and the GUI panel info.

Note that the sizers are in fitpage.py

**Comment by butlerpd** [comment]
Trac update at `2017/10/27 07:51:04`: 
* **butler** changed owner from "" to "butler"
* **butler** changed priority from "trivial" to "critical"
* **butler** changed status from "new" to "assigned"

**Comment by butlerpd** [comment]
Trac update at `2018/03/11 18:46:25`: **butler** commented:

This is now sasmodels pull request 63 and awaits verification before merging to master.  Once done this ticket can be closed.

**Comment by butlerpd** [comment]
Trac update at `2018/03/18 18:02:48`: **butler** commented:

In changeset 47ace500b69b84227cf32c9ef9bb1118c093b7e2:

```text
#!CommitTicketReference repository="sasview" revision="47ace500b69b84227cf32c9ef9bb1118c093b7e2"
fix as per comments.

addresses #1012
```

**Comment by butlerpd** [comment]
Trac update at `2018/03/19 21:17:25`: 
* **butler** commented:

the last changes were approved and the pull request merged but forgot to set automatically close with Closes #1012.  Closing now.


* **butler** changed resolution from "" to "fixed"
* **butler** changed status from "assigned" to "closed"
